app/controllers/record_controller.rb

```diff
@@ -0,0 +1,32 @@
+class RecordController < ApplicationController
+  before_action :validate_id!, only: %i[view]
+
+  include RecordHelper
+
+  def view
+    # If there are formatting requirements for a received ID value, they should be enforced here.
+    @id = params[:id]
+
+    timdex = TimdexWrapper.new
+    response = timdex.record(@id)
+
+    # Detection of unexpected response from the API would go here...
+
+    # Manipulation of returned record:
+    # The API includes three housekeeping fields which are not part of the record.
+    %w[request_limit request_count limit_info].each do |value|
```

Comment:
Having these fields in the response doesn't cause problematic behavior - I just don't see us displaying this information as part of the full record view, so figured it would be preferable to remove them. The alternative would be to leave them there, but just not render them at the view level - which might cause problems later if we render fields using an iterator (which, yes, is probably a bit of premature optimization).

Mostly this was a case of me trying to keep things out unless we actually need them. I'm happy to remove this, though, if you think removing it is more expensive in a processing budget than leaving them undisturbed.